02-intro-to-genomic-data.Rmd

```diff
@@ -5,10 +5,67 @@ Borrow from https://hutchdatascience.org/Choosing_Genomics_Tools/
 
 
 ## What is genomic data?
+Genome: Complete set of DNA in an organism.
 
+Gene: DNA segment with instructions for protein synthesis.
+
+DNA: Molecule carrying genetic instructions.
+
+Genomics: Study of genomes, including structure, function, and evolution.
+
+Sequence Data: Includes Whole Genome Sequencing (WGS) and Exome Sequencing (focuses on exons).
+
+Gene Expression Data: Transcriptomics (e.g., RNA-Seq) measures mRNA levels.
+
+Epigenetic Data: Studies chemical modifications like DNA methylation affecting gene expression.
+Variant Data: Highlights genetic differences (e.g., SNPs, Indels, Structural Variants).
+
+Biological Samples: Blood, tissue, saliva.
+
+Public Databases: GenBank, The Cancer Genome Atlas (TCGA)
+
+Medicine: Personalized medicine, disease diagnosis, and targeted therapies.
+
+Research: Understanding genetic diseases, drug discovery.
+
+Evolutionary Biology: Comparative genomics to study evolutionary relationships.
+
+Bioinformatics Software: BLAST (sequence alignment), STAR (RNA-Seq), GATK (variant calling).
+
+Statistical Methods and Visualization Tools: IGV for data visualization.
+
+
+Genomic data refers to the information obtained from studying the genomes of organisms, where a genome is the complete set of genetic material (DNA) in an organism, including all its genes. This data provides some insight into the structure, function, and evolution of genomes. Key terms in genomic data include the genome, which is the entire set of DNA including all its genes; a gene, which is a segment of DNA containing instructions for building proteins; DNA (Deoxyribonucleic Acid), the molecule that carries genetic instructions; and genomics, the study of genomes, including their structure, function, evolution, and mapping.
```

Comment:
Ah I see. It's my fault because the outline I wrote said to put the central dogma later. Now that I'm reading this I'm thinking that should come first. What do you think?